int32 x_req
int32 y_req
int32 robot_id

